File: Dddml.Wms.HttpServices.JavaClient/src/org/dddml/wms/domain/UserRoleId.java
package org.dddml.wms.domain;


public class UserRoleId
{
    private String userId;

    public String getUserId()
    {
        return this.userId;
    }

    public void setUserId(String userId)
    {
        this.userId = userId;
    }

    private String roleId;

    public String getRoleId()
    {
        return this.roleId;
    }

    public void setRoleId(String roleId)
    {
        this.roleId = roleId;
    }

    UserRoleId ()
    {
    }

    public UserRoleId (String userId, String roleId)
    {
        this.userId = userId;
        this.roleId = roleId;
    }

}

File: Dddml.Wms.AdminUI/ts/UserRoleId.ts
namespace Dddml.Wms.Domain {

	export class UserRoleId {

		private userId: string;

		getUserId(): string {
			return this.userId;
		}

		setUserId(userId: string) {
			this.userId = userId;
		}

		private roleId: string;

		getRoleId(): string {
			return this.roleId;
		}

		setRoleId(roleId: string) {
			this.roleId = roleId;
		}

	}
}

